students/12th/kyunghun/project_westagram/posting/views.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from django.views   import View
+from django.http    import JsonResponse
+
+from .models        import PostMedia, Photo
+from user.models    import Users
+    
+class PostingView(View):   
+    def post(self, request):
+            data                 = json.loads(request.body)
+            login_user           = Users.objects.get(email = data['email'])
+            login_user_id        = login_user.id
```

Comment:
```suggestion
            login_user_id           = Users.objects.get(email = data['email']).id
```
객체에 접근 할 때, 특정 객체가 정해지면 그 속성에 계속 `.`으로 연결하여 접근할 수 있습니다. 한 번 활용해보세요